app/services/foreman/plugin.rb

```diff
@@ -351,6 +351,11 @@ def parameter_filters(klass)
       @parameter_filters.fetch(klass.is_a?(Class) ? klass.name : klass, [])
     end
 
+    # Add concerns to a host
+    def extend_host(concern_module)
```

Comment:
We don't want plugin devs to touch internal Foreman objects, later we might just change how extend_host works without requiring plugin changes, imagine Host::Managed would change to Host::Base e.g.